sdk/python/feast/batch_feature_view.py

```diff
@@ -0,0 +1,61 @@
+from datetime import timedelta
+from typing import Dict, List, Optional, Union
+
+from google.protobuf.duration_pb2 import Duration
```

Comment:
Do we still need to support Duration? Isn't it a good time to deprecate it?

Reply:
yes it is. Removed